ksqldb-functional-tests/src/main/java/io/confluent/ksql/test/model/PostConditionsNode.java

```diff
@@ -133,7 +138,34 @@ Pattern buildBlackList() {
     @SuppressWarnings("unchecked")
     Matcher<Iterable<PostTopicNode>> buildTopics() {
       final Matcher<PostTopicNode>[] matchers = topics.stream()
-          .map(Matchers::is)
+          .map(topic -> new BaseMatcher<PostTopicNode>() {
```

Comment:
nit: can we move this matcher logic into `PostTopicNode`, so someone updating `PostTopicNode` is less likely to forget to update the matcher? The other QTT classes all follow a particular pattern with this, for example: https://github.com/confluentinc/ksql/blob/master/ksqldb-functional-tests/src/main/java/io/confluent/ksql/test/model/SourceNode.java#L114